docs/DafnyRef/Types.md

```diff
@@ -1274,7 +1274,7 @@ type.  For such assignments, an explicit conversion must be used, see
 
 Dafny builds in three families of subset types, as described next.
 
-### 11.3.1 Type`nat`
+### 11.3.1.1 Type`nat`
```

Comment:
?

Reply:
I think this was done by one of the packaging scripts... Looking at it now, I don't think it should be included.

Reply:
There's some fancy regex magic going on somewhere :)